Package.swift

```diff
@@ -44,23 +44,23 @@ let package = Package(
     .package(
       name: "AppAuth",
       url: "https://github.com/openid/AppAuth-iOS.git",
-      "1.6.0" ..< "2.0.0"),
+      from: "1.7.0"),
```

Comment:
note: opted to switch to the `from: ...` rule since it's the recommended way according to the Swift [docs](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#package-dependency), and is equivalent to the explicit ranges we were using